windows-observ-lib/jsonnetfile.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/jsonnet-libs.git",
+          "subdir": "common-lib"
+        }
+      },
+      "version": "vzhuravlev/common-lib"
```

Comment:
Should this get merged first? Given it's a requirement, we should probably have that nailed down with some certainty that the interface will not change.

Reply:
commonlib is ready for review. once we merge it, we can continue with windows lib.